src/Microsoft.Identity.Web/MicrosoftGraph/MicrosoftGraphServiceExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Graph;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Extensions methods on a MicrososoftAppCallingWebApiAuthenticationBuilder builder
+    /// to add support to call Microsoft Graph.
+    /// </summary>
+    public static class MicrosoftGraphServiceExtensions
+    {
+        /// <summary>
+        /// Add support to calls Microsoft graph. From a named option and a configuration section.
+        /// </summary>
+        /// <param name="builder">Builder.</param>
+        /// <param name="configurationSection">Configuraiton section.</param>
+        /// <returns>The builder to chain.</returns>
+        public static MicrosoftIdentityAppCallsWebApiAuthenticationBuilder AddMicrosoftGraphServiceClient(
+            this MicrosoftIdentityAppCallsWebApiAuthenticationBuilder builder,
+            IConfigurationSection configurationSection)
+        {
+            return builder.AddMicrosoftGraphServiceClient(
+                options => configurationSection.Bind(options));
+        }
+
+        /// <summary>
+        /// Add support to calls Microsoft graph. From a base graph Url and a default scope.
+        /// </summary>
+        /// <param name="builder">Builder.</param>
+        /// <param name="graphBaseUrl">Named instance of option.</param>
+        /// <param name="defaultScopes">Configuraiton section.</param>
+        /// <returns>The builder to chain.</returns>
+        public static MicrosoftIdentityAppCallsWebApiAuthenticationBuilder AddMicrosoftGraphServiceClient(
+            this MicrosoftIdentityAppCallsWebApiAuthenticationBuilder builder,
+            string graphBaseUrl = "https://graph.microsoft.com/v1.0",
+            string defaultScopes = "user.read")
+        {
+            return builder.AddMicrosoftGraphServiceClient(
+                options =>
+                {
+                    options.BaseUrl = graphBaseUrl;
+                    options.Scopes = defaultScopes;
+                });
+        }
+
+        /// <summary>
+        /// Add support to calls Microsoft graph. From a named options and a configuraiton method.
```

Comment:
```suggestion
        /// Add support to call Microsoft Graph. From a named options and a configuration method.
```

Reply:
fixed in this [PR](https://github.com/AzureAD/microsoft-identity-web/pull/442)

---
In reply to: [469486903](https://github.com/AzureAD/microsoft-identity-web/pull/434#discussion_r469486903) [](ancestors = 469486903)